sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.NUnit/samples/customising-service-parameters/README.md

```diff
@@ -0,0 +1,128 @@
+## Learn about different available service parameters and how to use them
+
+Follow the steps listed in this [README](../../README.md) to integrate your existing Playwright test suite with the Microsoft Playwright Testing service.
+
+This guide explains the different options available to you in the Azure.Developer.MicrosoftPlaywrightTesting.NUnit package and how to use them.
+
+### Using runsettings file
+
+1. Create a `.runsettings` file in the root directory
+
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <RunSettings>
+        <TestRunParameters>
+            <!-- The below parameters are optional -->
+            <Parameter name="Os" value="linux" />
+            <Parameter name="RunId" value="sample-run-id" />
+            <Parameter name="ExposeNetwork" value="<loopback>" />
+            <Parameter name="DefaultAuth" value="ENTRA" />
+            <Parameter name="UseCloudHostedBrowsers" value="true" />
+            <Parameter name="AzureTokenCredentialType" value="DefaultAzureCredential" />
+            <Parameter name="ManagedIdentityClientId" value="77bfc267-86cb-4eeb-9e4a-747a217a318c" />
+            <Parameter name="EnableGitHubSummary" value="false" />
+        </TestRunParameters>
+        <!-- Enable Reporting feature -->
+        <LoggerRunSettings>
+            <Loggers>
+                <Logger friendlyName="ms-playwright-service" enabled="true" />
+            </Loggers>
+        </LoggerRunSettings>
+    </RunSettings>
+    ```
+
+  > [!NOTE]
+  > You can also modify the runid by setting the environment variable `PLAYWRIGHT_SERVICE_RUN_ID`.
+
+2. Run tests using the above runsettings file
+
+    ```dotnetcli
+    dotnet test --settings .runsettings
+    ```
+
+#### Known Issue - Minimal support for Azure Credentials
+
+This issue only impacts the reporting feature. Currently, the service provides minimal support for the following [Azure Credential types](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/identity-readme?view=azure-dotnet)
```

Comment:
I assume all Azure Identity library credentials are affected. If so, link here instead: https://learn.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet#credential-classes. Then you can remove this hard-coded list.